src/NuGet.Core/NuGet.Frameworks/NuGetFramework.cs

```diff
@@ -77,6 +119,32 @@ public Version Version
             get { return _frameworkVersion; }
         }
 
+        /// <summary>
+        /// Framework Platform (net5.0+)
+        /// </summary>
+        public string Platform
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// Framework Platform Version (net5.0+)
+        /// </summary>
+        public Version PlatformVersion
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// True if the platform is non-empty
+        /// </summary>
+        public bool HasPlatform
+        {
```

Comment:
Can we use more modern syntax here? `public bool HasPlatform => !string.IsNullOrEmpty(Platform);`

Reply:
I'd rather keep things consistent with the rest of the file, and I think it's out of scope to change all the other ones that work this way.